flox-file-parser/app/lib/parser.js

```diff
@@ -1,139 +1,106 @@
 const fs = require("fs")
 const path = require("path")
+const db = require("../../database/models")
 
 const videoNameParser = require("video-name-parser")
 
 const supportedVideoFileTypes = ["mkv", "mp4"]
 const env = process.env
```

Comment:
Seems we don't use `env`, `supportedVideoFileTypes` and `videoNameParser` in this file.